analysis.js

```diff
@@ -2,70 +2,166 @@ $("#chart-weeklyDatePicker").datetimepicker({
   format: 'YYYY-MM-DD'
```

Comment:
this function do nothing, because `#chart-weeklyDatePicker` is renamed to `"#chart-weeklyDatePickerStart`